esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3153,8 +3176,35 @@ bool CFileSprayEx::onDropZoneFileSearch(IEspContext &context, IEspDropZoneFileSe
         if (!validNameFilter)
             throw MakeStringException(ECLWATCH_INVALID_INPUT, "Invalid Name Filter '*'");
 
+        double version = context.getClientVersion();
```

Comment:
Should the dropZoneServerReq be allowed to be null if there is only one server associated with the drop zone (likely to be very common)?
It could also mean all servers for a given name.

Reply:
The code is not from this PR. I think it is a good enhancement to allow dropZoneServerReq be null. Changed it in another PR?